Update README

weiliw-amz opened the pull request:
*Issue #, if available:*
N/A

*Description of changes:*
Updated README for v0.2.0 release.
Fixed `python` --> `python3` commands in guides and scripts. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

weiliw-amz committed:
Update README

rofuyu commented on README.md:
```diff
@@ -22,6 +22,11 @@ Given an input, PECOS identifies a small set (10-100) of relevant outputs from a
 
 * text2text application ([`pecos.apps.text2text`](pecos/apps/text2text/README.md)): an easy-to-use text classification pipeline (with X-Linear backend) that supports n-gram TFIDF vectorization, classification, and ensemble predictions. 
```
let's remove "classification, and ensemble prediction." from this.

OctoberChang replied:
For the `X-Transformer` subsection, i would suggest to add the latest XR-Transformer citation as well.

weiliw-amz replied:
> let's remove "classification, and ensemble prediction." from this.

Removed.

weiliw-amz replied:
> For the `X-Transformer` subsection, i would suggest to add the latest XR-Transformer citation as well.

@jiong-zhang has a few entries to update for README. He will open a new PR and make the modifications together.

weiliw-amz committed:
Add sudo to README commands

weiliw-amz commented:
Added `sudo` to commands. Also added that the commands only work for Amazon Linux 2 Images. For EC2 Amazon Linux 2 AMI, some of the packages does not need to be installed.
